TSC-Charter.md

```diff
@@ -126,8 +137,8 @@ representatives of the community.
 For election of persons (TSC Chairperson, Maintainers, etc.) a
 multiple-candidate method should be used, e.g.:
 
-* [Condorcet](http://en.wikipedia.org/wiki/Condorcet_method) or
-* [Single Transferable Vote](http://en.wikipedia.org/wiki/Single_transferable_vote)
+* [Condorcet][] or
```

Comment:
Oh cool, I didn't realize you could leave the second entry blank and it would Just Work